cmd/cosign/cli/verify.go

```diff
@@ -87,17 +92,20 @@ func (c *VerifyCommand) Exec(ctx context.Context, args []string) error {
 	if len(args) == 0 {
 		return flag.ErrHelp
 	}
-	if !oneOf(c.Key, c.KmsVal) {
+	if !oneOf(c.Key, c.KeyURL, c.KmsVal) {
```

Comment:
wdyt about trying to handle this transparently in the same flag, like kubectl does?

Reply:
works for me

Reply:
done